### ISSUE TITLE
Unusable since Neovim 0.8

### ISSUE THREAD

**Issue by yuki-uthman** [google/vroom#124]
Stops working after upgrading to 0.8 with the following Error.

Basically it gets stuck after neovim opens and after a while it outputs this error msg.

```python
ERROR: Traceback (most recent call last):
  File "/Users/Yuki/.asdf/installs/python/3.9.16/lib/python3.9/site-packages/vroom/runner.py", line 74, in __call__
    self.env.vim.Start()
  File "/Users/Yuki/.asdf/installs/python/3.9.16/lib/python3.9/site-packages/vroom/neovim_mod.py", line 38, in Start
    self.nvim = neovim.attach('socket', path=self.args.servername)
  File "/Users/Yuki/.asdf/installs/python/3.9.16/lib/python3.9/site-packages/pynvim/__init__.py", line 111, in attach
    socket_session(path) if session_type == 'socket' else
  File "/Users/Yuki/.asdf/installs/python/3.9.16/lib/python3.9/site-packages/pynvim/msgpack_rpc/__init__.py", line 35, in socket_session
    return session('socket', path)
  File "/Users/Yuki/.asdf/installs/python/3.9.16/lib/python3.9/site-packages/pynvim/msgpack_rpc/__init__.py", line 19, in session
    loop = EventLoop(transport_type, *args, **kwargs)
  File "/Users/Yuki/.asdf/installs/python/3.9.16/lib/python3.9/site-packages/pynvim/msgpack_rpc/event_loop/base.py", line 93, in __init__
    raise e
  File "/Users/Yuki/.asdf/installs/python/3.9.16/lib/python3.9/site-packages/pynvim/msgpack_rpc/event_loop/base.py", line 90, in __init__
    getattr(self, '_connect_{}'.format(transport_type))(*args)
  File "/Users/Yuki/.asdf/installs/python/3.9.16/lib/python3.9/site-packages/pynvim/msgpack_rpc/event_loop/asyncio.py", line 96, in _connect_socket
    self._loop.run_until_complete(coroutine)
  File "/Users/Yuki/.asdf/installs/python/3.9.16/lib/python3.9/asyncio/base_events.py", line 647, in run_until_complete
    return future.result()
  File "/Users/Yuki/.asdf/installs/python/3.9.16/lib/python3.9/asyncio/unix_events.py", line 244, in create_unix_connection
    await self.sock_connect(sock, path)
  File "/Users/Yuki/.asdf/installs/python/3.9.16/lib/python3.9/asyncio/selector_events.py", line 500, in sock_connect
    return await fut
  File "/Users/Yuki/.asdf/installs/python/3.9.16/lib/python3.9/asyncio/selector_events.py", line 505, in _sock_connect
 Traceback (most recent call last):

   sock.connect(address)
FileNotFoundError: [Errno 2] No such file or directory
```

Absolutely love this tester.
Hope someone can fix it....

**Comment by dbarnett** [comment]
Strange, all the vroom code is doing there is asking pynvim to attach, nothing manual with socket files. I wonder if it's a bug between pynvim and neovim. Have you tried checking if you can upgrade pynvim yet?

**Comment by yuki-uthman** [comment]
Yeah tried it with the latest version of pynvim 0.4.3.
But have not tried installing it from the master branch. 

Everything works fine until neovim 0.7.2.

**Comment by dbarnett** [comment]
Well, two angles would be trying to bisect down to which nvim change broke it or to boil down a minimal repro case against neovim (w/o involving vroom) and submit upstream.

@tarruda you wouldn't have any idea offhand, would you?

**Comment by dbarnett** [comment]
No word back from Tiago. @yuki-uthman just to confirm: you're still stuck on this?

**Comment by yuki-uthman** [comment]
yeah, I just checked with 0.8.3 as well just in case.
Still the same error.

**Comment by dbarnett** [comment]
Hey, I figured out the problem, and it turned out to be a pretty simple fix! Feel free to try out PR #131 in the meantime while waiting for it to get reviewed and merged.
